package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1020,7 +1020,8 @@ def shape_parameter(group, **kwargs):
         ('shape_parameter', shape_parameter))
 
     @warn_if_not_unique
-    def asphericity(group, pbc=None):
+    @check_pbc_and_unwrap
+    def asphericity(group, pbc=None, unwrap=None, compound='group'):
```

Comment:
You're adding a `compound` kwarg here with no tests, it's probably best not to add it now

Reply:
I have added tests for compound now.